src/modules/game/activities/blacksmith/modal/blacksmith-modal.component.html

```diff
@@ -117,3 +153,11 @@
     >
   </div>
 </ng-template>
+
+<ng-template #magicDustExplanation>
+  <div class="p-2">
+    <span class="text-white bg-black"
+      >Using the magic dust will avoid the item from breaking!</span
```

Comment:
Using the magic dust will prevent the item from breaking!

Avoid ahí queda un poco raro 😄

Reply:
https://www.youtube.com/shorts/kRL7sBXh0_o